### PR TITLE
Zpětnovazebník

### PR DESCRIPTION
**Pull request by honzajavorek**
Added

- `https://pyvo.cz/zpetnovazebnik` and `https://pyvo.cz/feedback` redirects
- notice on every page that people should fill the form

screen shot 2017-01-26 at 10 06 27

### CODE REVIEW

**Comment by aleszoulek** [comment]
+1

File: pyvocz/views.py [hunk 1]
```diff
@@ -317,3 +317,12 @@ def subdomain_redirect(subdomain):
     }
     subdomain = ALIASES.get(subdomain, subdomain)
     return redirect(url_for('series', series_slug=subdomain))
+
+@route('/feedback')
+@route('/zpetnovazebnik')
```
Comment (encukou) [inline pyvocz/views.py]: Why the two ways to do it?

**Comment by encukou** [comment]
How long should this be there?

Can we use a less aggressive color, like #c58c11?

**Comment by honzajavorek** [comment]
@encukou in that case, I'd use pythonic blue. It should be vivid enough to grab attention, what do you think?

Regarding the URLs, I don't mind which one we use as a canonical one, but then we need to update https://github.com/pyvec/python.cz/pull/181 to be aligned. (Which is the same case for the color, btw)

**Comment by honzajavorek** [comment]
pythonic blue: `#337ab7`-ish

**Comment by encukou** [comment]
Deployed and live.

**Pull request merged**